scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -10,6 +10,9 @@ workflow GvsJointVariantCalling {
         Array[File] input_vcfs
         Array[File] input_vcf_indexes
         String call_set_identifier
+        File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+        Boolean use_interval_weights = true
+        File interval_weights_bed = "gs://broad-public-datasets/gvs/weights/gvs_vet_weights_1kb.bed"
         String? extract_output_gcs_dir
         String drop_state = "FORTY"
```

Comment:
Do we know that the exomes have been reblocked with the latest version? Or should this be 60?